hydra/beam_sampler.py

```diff
@@ -6,6 +6,7 @@
 
 from pyuvsim import AnalyticBeam
 from pyuvsim.analyticbeam import diameter_to_sigma
+from pyuvdata import 
```

Comment:
Do you want to keep this from statement?

Reply:
lol